src/libraries/System.Runtime/tests/System.Globalization.Tests/CompareInfo/CompareInfoTests.Compare.cs

```diff
@@ -536,15 +523,15 @@ public void TestIgnoreKanaAndWidthCases()
 
             // Edge case of the Ignore Width.
             Assert.False(string.Compare("\u3162\u3163", "\uFFDB\uFFDC", CultureInfo.InvariantCulture, CompareOptions.None) == 0, $"Expect '0x3162 0x3163' != '0xFFDB 0xFFDC'");
-            if (!PlatformDetection.IsHybridGlobalizationOnBrowser)
+            if (!PlatformDetection.IsHybridGlobalizationOnBrowser && !PlatformDetection.IsHybridGlobalizationOnOSX)
                 Assert.True(string.Compare("\u3162\u3163", "\uFFDB\uFFDC", CultureInfo.InvariantCulture, CompareOptions.IgnoreWidth) == 0, "Expect '0x3162 0x3163' == '0xFFDB 0xFFDC'");
 
             const char hiraganaStart = '\u3041';
             const char hiraganaEnd = '\u3096';
             const int hiraganaToKatakanaOffset = 0x30a1 - 0x3041;
 
             // in HybridGlobalization on Browser IgnoreKanaType is supported only for "ja-JP"
-            CultureInfo ignoreKanaTypeTestedCulture = PlatformDetection.IsHybridGlobalizationOnBrowser ? new CultureInfo("ja-JP") : CultureInfo.InvariantCulture;
+            CultureInfo ignoreKanaTypeTestedCulture = PlatformDetection.IsHybridGlobalization ? new CultureInfo("ja-JP") : CultureInfo.InvariantCulture;
```

Comment:
is this change required? according to the comment we only need to use ja-JP on browser

Reply:
It is, as this test case fails when using `InvariantCulture` on `iOS` also. But for iOS `IgnoreKanaType` is supported in other cultures, there are other test cases that pass.
I will add comment for iOS that this particular test case is behaving different.